manim/mobject/probability.py

```diff
@@ -2,7 +2,11 @@
 
 __all__ = ["SampleSpace", "BarChart"]
 
+
+from typing import Iterable, List
+
 import numpy as np
+from colour import Color
```

Comment:
did you import this module just for typings?

Reply:
I think what @naveen521kk wants to point out is that you make the import like here:
https://github.com/ManimCommunity/manim/blob/fe254fc241e0db59c8497d8b7b01d3d3d3774474/manim/animation/update.py#L11-L12
Also, this information should be put here:
https://docs.manim.community/en/stable/contributing/typings.html
@Nikhil0504 : can you maybe update this typings section in the docs to add that information?